alf/algorithms/td_loss.py

```diff
@@ -91,6 +97,8 @@ def forward(self, experience, value, target_value):
             target_value (torch.Tensor): the time-major tensor for the value at
                 each time step. This is used to calculate return. ``target_value``
                 can be same as ``value``.
+            train_info (sarsa info, sac info): information used to calcuate importance_ratio
```

Comment:
What is sarsa info, sac info here? Can this function be used with other algorithms beyond sac and sarsa?